### PR TITLE
Added Cards

### PR DESCRIPTION
**Pull request by sri-rishi**
Cards

- Horizontal cards
- Vertical cards
- Text Overlay cards
- Text only cards

### CODE REVIEW

File: Docs/Components_html/cards.html [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+        <main class="main-section flex-column">
+            <section>
+                <h1 class="headings main-heading font-weight-4">Cards</h1>
+                <p class="heading-about font-weight-3"> 
+                    Card are used to show user related data collectively, like product details.
+                </p>
+            </section>
+            <section class="component-section flex-column">
+                <div>
+                    <h2 class="heading sub-heading font-weight-4">
```
Comment (Ankur9669) [inline Docs/Components_html/cards.html:69]: I think this div tag could be removed

File: lib/Components/cards.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    font-size: 2rem;
+    color: var(--gray-color);
+    padding: 10px 1rem;
+    border-radius: inherit;
+}
+
+.card-vl-text {
+    max-width: 30rem;
+    padding: 20px;
+}
```
Comment (Ankur9669) [inline lib/Components/cards.css:141]: Could use rem here, but that is only a suggestion because padding can be given in px

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
image
On adding text to card it is breaking.

Overall the UI looks great, but the cards are not responsive, so you can have problem using these in your future projects

File: Docs/Components_html/cards.html (hunk 1 above)
Comment (sri-rishi) [reply Docs/Components_html/cards.html:69]: div here is to separate and giving gap between sub-heading and component section

**Comment by sri-rishi** [comment]
> image On adding text to card it is breaking.
> 
> Overall the UI looks great, but the cards are not responsive, so you can have problem using these in your future projects

Changes are done you can check again and fixed them

File: lib/Components/cards.css (hunk 2 above)
Comment (sri-rishi) [reply lib/Components/cards.css:141]: Actually I  don't want to change padding for different viewport size

**Comment by varun-Tad** [comment]
Looks good to merge 👍

**Pull request merged**